src/props.sizes.js

```diff
@@ -43,4 +43,21 @@ export default {
   '--size-lg': '1024px',
   '--size-xl': '1440px',
   '--size-xxl': '1920px',
+
+  '--size-relative-0': '.25ch',
+  '--size-relative-1': '.5ch',
+  '--size-relative-2': '.75ch',
+  '--size-relative-3': '1ch',
+  '--size-relative-4': '1.25ch',
+  '--size-relative-5': '1.5ch',
+  '--size-relative-6': '1.75ch',
+  '--size-relative-7': '2ch',
+  '--size-relative-8': '3ch',
+  '--size-relative-9': '4ch',
+  '--size-relative-10': '5ch',
+  '--size-relative-11': '6ch',
+  '--size-relative-12': '7ch',
+  '--size-relative-13': '8ch',
+  '--size-relative-14': '10ch',
+  '--size-relative-15': '12ch',
```

Comment:
I wonder if this list should be the exact same as the rem sizes but just swapped to the `ch` unit? it'd make interchanging between them easier AND the mental model of the scale easier. We'd also gain the 2 negative values. So like, less to manage and more consistent? thoughts?